thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -90,6 +96,7 @@ type EvalFnRunner struct {
 	OutContent           bytes.Buffer
 	FromStdin            bool
 	Image                string
+	AddImage             bool
```

Comment:
Can we rename it to something like `SaveToKptfile` or `AddToPipeline` or `SaveCommand` to reflect the usage better.

Because if user use `--exec`, we will save that as well to the `Kptfile`, right ?

Reply:
It won't be saved if users use `--exec`. I added a check in `preRunE` to make sure this flag only works with `--image` flag.

Renamed.